test/e2e/e2e_test.go

```diff
@@ -44,7 +44,7 @@ import (
 
 var _ = Describe("controller", Ordered, func() {
 	var namespace string = "instaslice-system"
-	var defaultNamespace string = "default"
+	var defaultNamespace string = namespace
```

Comment:
Can you use a variable for the namespace?

Reply:
I dont understand, we are using variable here

Reply:
I see you only want one var, ok will do.

Reply:
You don't use this variable in the tests. You use the hard coded "instaslice-system".

Reply:
https://github.com/openshift/instaslice-operator/pull/194/files#r1813144739